app/hotels/src/gallery/PhotosStripe.js

```diff
@@ -14,10 +14,28 @@ type Props = {|
   onClose: () => void,
 |};
 
-export default class PhotosStripe extends React.Component<Props> {
+type State = {|
+  key: number,
+|};
+
+/* 
+  Need to trigger a re-render on android to show photo
+  https://github.com/leecade/react-native-swiper/issues/227
+*/
+export default class PhotosStripe extends React.Component<Props, State> {
+  state = {
+    key: Math.random(),
+  };
+
+  componentDidMount = () => {
+    if (Platform.OS === 'android') {
+      setTimeout(() => this.setState({ key: Math.random() }));
+    }
+  };
+
   renderPagination = (index: number, total: number) => {
     return (
-      <View style={{ position: 'absolute', top: 20, width: '100%' }}>
+      <View style={{ position: 'absolute', top: 40, width: '100%' }}>
```

Comment:
Is this good solution? This is why I suggested `SafeAreaView` because now it looks really weird on normal devices... 🙂

Reply:
Ok, I can see if we can make it work with SafeAreaView, I will remove it from this PR.
SafeAreaView will not work with `position: 'absolute'`

Reply:
Leave it if not possible - it's just that the behavior is much more sophisticated... :)

Reply:
Yeah, I will leave it out of this PR, and make a new one for the Header on iphone X